.gitlab-ci.yml

```diff
@@ -282,6 +282,37 @@ cwl_v1.2:
     when: always
     expire_in: 14 day
 
+cwl_badge:
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+    - if: $CI_COMMIT_TAG
+    - if: $CI_COMMIT_BRANCH =~ /.*-fix-ci/
+    - if: $CI_COMMIT_BRANCH
+      changes:
+        compare_to: 'refs/heads/master'
+        paths:
+          - 'src/toil/cwl/*'
+          - 'src/toil/test/cwl/*'
+  stage: integration
+  script:
+    - pwd
+    - ${MAIN_PYTHON_PKG} -m virtualenv venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[cwl,aws]
+    - python setup_gitlab_docker.py  # login to increase the docker.io rate limit
+    # Run CWL integration tests excluded from cwl_misc
+    - make download_cwl_spec
+    - rm -rf badges
+    # We have to run this on one thread to avoid a race condition: https://github.com/common-workflow-language/cwltest/issues/239
+    - make test tests="src/toil/test/cwl/spec_v12/conformance_tests.cwltest.yaml" dist=worksteal threads="1" pytest_args="--cwl-runner-verbose --cwl-badgedir=badges --junit-xml=in-place-update-conformance-1.2.junit.xml --cwl-args='--relax-path-checks --clean=always --logDebug --statusWait=10 --retryCount=2 --eval-timeout=600'"
```

Comment:
Actually we might consider butting the 1.2 results in `badges1.2` so we can point to conformance to a particular spec version, even if we can't follow the particular branch.